scripts/blueprint_manager.gd

```diff
@@ -35,6 +35,9 @@ func _process(delta: float) -> void:
 	
 	var blueprints: Array = Blueprint.get_all()
 	blueprints.sort_custom(func(a: Blueprint, b: Blueprint) -> bool:
+		if not a.id or not b.id:
+			return true
+		
```

Comment:
I'm not sure if this is necessary but it was giving me a lot of errors (though adding this didn't fix it).

Try to remove and see if it works.